pkg/base-dev/sur/hood.hoon

```diff
@@ -138,10 +144,13 @@
     |=  [=weft =tape]
     (welp " {<[lal num]:weft>}" tape)
   ?.  verb
-    =/  base-hash  ?.(=(1 (lent meb)) <meb> <(head meb)>)
+    =/  cut=(list tape)  (turn meb truncate-hash)
+    =/  base-hash
+      ?:  =(1 (lent cut))  (head cut)
+        "~[{`tape`(zing (join " " `(list tape)`cut))}]"
```

Comment:
I would also format the case were the base hash is null. Otherwise it will print it as `~[]`.
```suggestion
    =/  len   (lent cut)
    =/  base-hash
      ?:  =(0 len)  "~"
      ?:  =(1 len)  (head cut)
        "~[{`tape`(zing (join " " `(list tape)`cut))}]"
```

Reply:
thx, added